contracts/PvpArena.sol

```diff
@@ -1009,7 +1010,8 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
 
         int128 bonusShieldStats;
         if (fighter.useShield) {
-            bonusShieldStats = _getShieldStats(character.ID);
+            // we set bonus shield stats as 0.2
+            bonusShieldStats = _getShieldStats(character.ID).mul(20).div(100);
```

Comment:
These weapon/shield multiplier values are 1-based, meaning the stats will add onto 1, not 0.
Since we will be adding the shield multiplier on top of the weapon, we need to counteract the base value of 1 to only add the stats, otherwise shields will still be too powerful (even a 1star shield with a single stat of 4 would give you +101% on your base character power).
The .sub(1) call would have to be done before the mul(20) step.

This is a very small change so I can handle it right before deployment

Reply:
Perfect, understood Dan. Thanks for the correction!